frontend/src/locales/de.json

```diff
@@ -99,6 +100,18 @@
     },
     "your_amount": "Dein Betrag"
   },
+  "gdd_per_link": {
+    "copy": "kopieren",
+    "created": "Der Link wurde erstellt!",
+    "header": "Gradidos versenden per Link",
+    "link-copied": "Link wurde in die Zwischenablage kopiert",
+    "not-copied": "Konnte den Link ncht kopieren: {err}",
+    "sentence_1": "Wähle einen Betrag, aus welchen du per Link versenden möchtest. Du kannst auch noch eine Nachricht eintragen. Beim Klick „jetzt generieren“ wird ein Link erstellt, den du versenden kannst.",
+    "sentence_2": "Der Link ist 14 Tage gültig!",
+    "sentence_3": "Wird der Link nicht innerhalb der 14 Tage Frist abgerufen, wird er automatisch gelöscht. Der Betrag wird dann wieder deinem Konto gutgeschrieben.",
+    "sentence_4": "Wer den Link klick aktiviert, erhält die Zahlung von deinem Konto. Wer noch nicht Mitglied bei Gradido ist, wird durch den Registrierungsprozess geleitet und bekommt den GDD Betrag nach Registrierung / Bestätigung seines Gradido Kontos gut geschrieben.",
+    "sentence_5": "Der Vergänglichkeitsbetrag wird von dir getragen und für die maximale Gültigkeit deinem Betrag aufgerechnet. Dir wird aber nur der Betrag als Vergänglichkeit berechnet, je nachdem wie viel Tage bis zum Einlösen des Links vergangen sind. Der Restbetrag wird dir wieder zurückerstattet."
+  },
```

Comment:
Please use `yarn i18n:report` and remove stuff once we finish this

Reply:
seperated issues pleace